plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueConverter.java

```diff
@@ -304,7 +304,7 @@ private static Storage fromGlueStorage(StorageDescriptor sd, String tablePartiti
 
         SerDeInfo serdeInfo = requireNonNull(sd.serdeInfo(), () -> "StorageDescriptor SerDeInfo is null: " + tablePartitionName);
         return new Storage(
-                StorageFormat.createNullable(serdeInfo.serializationLibrary(), sd.inputFormat(), sd.outputFormat()),
+                StorageFormat.createNullable(serdeInfo.serializationLibrary(), sd.inputFormat(), sd.outputFormat(), null),
```

Comment:
is there a way to get the file format from the `serdeInfo` or `sd` instead of `null` ?